.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: daily
```

Comment:
While I like the idea of doing github-actions updates, I think our plan is to mostly control this from `prometheus/prometheus` and use our repo sync updates to manage things.

```suggestion
```